aioesphomeapi/model.py

```diff
@@ -218,8 +232,10 @@ class ClimateInfo(EntityInfo):
 
 @attr.s
 class ClimateState(EntityState):
-    mode = attr.ib(type=ClimateMode, converter=ClimateMode, default=ClimateMode.OFF)
-    action = attr.ib(type=ClimateAction, converter=ClimateAction, default=ClimateAction.OFF)
+    mode = attr.ib(type=ClimateMode, converter=ClimateMode,
+                   default=ClimateMode.OFF)
+    action = attr.ib(type=ClimateAction, converter=ClimateAction,
+                     default=ClimateAction.OFF)
```

Comment:
I believe you need to add the new attributes (`fan_mode`) here.

Same for `supported_fan_modes` in the ClimateInfo object.